DOCS-12445-table-names-v4.4-cv (#2109)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/creating-and-managing-a-cluster-manually.adoc

Please note that enabling JMX might cause some performance overheads given that the underlying structure adds listeners to get the statistics for each individual node.

[[table-name-transformers]]
===== Table Name Transformers

The `mule.cluster.jdbcstoretableNametransformerstrategy` property enables you to define a custom transformer to modify table names.

> For example, if you set the following property in the `mule-cluster.properties` file:
> [source]
> ----
> mule.cluster.jdbcstoretableNametransformerstrategy=com.mulesoft.mule.runtime.module.cluster.api.persistence.query.MD5TableNameTransformerStrategy
> ----
> Mule hashes table names using MD5 and a prefix to identify that these are Mule tables. Hashing table names guarantees that the length constraint is honored.

Reviewer: Rework to avoid breaking the sentence around the example:
```suggestion
For example, if you set the following property in the `mule-cluster.properties` file, Mule hashes table names using MD5 and a prefix to identify tables as Mule tables:
[source]
----
mule.cluster.jdbcstoretableNametransformerstrategy=com.mulesoft.mule.runtime.module.cluster.api.persistence.query.MD5TableNameTransformerStrategy
----
Hashing table names guarantees that the length constraint is honored.
```

> Optionally, you can create a custom transformer strategy by implementing the `com.mulesoft.mule.runtime.module.cluster.api.persistence.query.TableNameTransformerStrategy` interface.

Reviewer: I think we mean "Alternatively," rather than "Optionally,"
Do we want to state that it is easier to set the property in the file rather than in the interface, or does it matter?
